TownSuite.TwainScanner/MainFrame.cs

```diff
@@ -131,27 +131,12 @@ private void MainFrame_Load(object sender, EventArgs e)
 
         private void DeleteFiles()
         {
-
-            if (string.Equals(DirText, System.IO.Path.Combine(Environment.GetEnvironmentVariable("TMP"), "TownSuite", "TwainScanner"), 
-                StringComparison.InvariantCultureIgnoreCase))
-            {
-                // using a custom folder.   Delete everything.
-                LoopFiles("*.bmp");
-                LoopFiles("*.jpeg");
-                LoopFiles("*.tif");
-                LoopFiles("*.png");
-                LoopFiles("*.pdf");
-                LoopFiles("*.txt");
-            }
-            else
-            {
-                LoopFiles("tmpscan*.bmp");
-                LoopFiles("tmpscan*.jpeg");
-                LoopFiles("tmpscan*.tif");
-                LoopFiles("tmpscan*.png");
-                LoopFiles("tmpscan*.pdf");
-                LoopFiles("tmpscan*.txt");
-            }
+            LoopFiles("tmpScan.bmp");
+            LoopFiles("tmpScan.jpeg");
+            LoopFiles("tmpScan.tif");
+            LoopFiles("tmpScan.png");
+            LoopFiles("tmpScan.pdf");
+            LoopFiles("tmpScan.txt");

```

Comment:
This leaves temp files hanging around on the file system

![image](https://github.com/TownSuite/TownSuite.TwainScanner/assets/656288/14a56363-1d98-413a-afe3-ef289d9bd8cd)